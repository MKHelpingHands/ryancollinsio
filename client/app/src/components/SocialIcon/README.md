## SocialIcon Component
A component that ...

### Example

```js
<SocialIcon />
```

### Props

| Prop          | Type     | Default     | Possible Values
| ------------- | -------- | ----------- | ---------------------------------------------
| **myProp**    | String   |             | Any string value


### Other Information
