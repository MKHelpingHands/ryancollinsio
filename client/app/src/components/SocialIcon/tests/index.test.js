import SocialIcon from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<SocialIcon />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <SocialIcon />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
