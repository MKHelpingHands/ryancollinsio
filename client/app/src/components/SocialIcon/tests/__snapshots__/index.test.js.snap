exports[`<SocialIcon /> should render with default props 1`] = `<span />`;
